reflex-algos/components/PySpark/hello_world/hello_world.py

```diff
@@ -0,0 +1,40 @@
+""" Hello World.
+Prints "Hello, world."
+"""
+
+from __future__ import print_function
+
+from pyspark import SparkContext
+
+import argparse
+import datetime
+import random
+
+def main():
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--use_mlops', type=int, dest='use_mlops', default=0)
+    parser.add_argument("--echo_arg", dest='echo_arg', help="Program will print this argument", default="Hello, world")
+    args = parser.parse_args()
+
+    sc = SparkContext(appName="hello-world")
+    if args.use_mlops > 0:
```

Comment:
use mlops in optional argument in component.json so we must add some else condition here.

Reply:
Not sure I follow this. It has a default value to not use mlops.